data_prep/scotland/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "data_prep"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+anyhow = "1.0.82"
+backend = { path = "../../backend" }
+bincode = "1.3.3"
+csv = "1.3.0"
+geo = "0.29.1"
+geojson = { git = "https://github.com/georust/geojson", features = ["geo-types"] }
+serde = "1.0.188"
+utils = { git = "https://github.com/a-b-street/utils" }
+
+[patch.crates-io]
+# use unreleased `geo` and `geo-types` to fix crash: https://github.com/a-b-street/ltn/issues/54
+# (awaiting release of https://github.com/georust/geo/pull/1279)
+geo = {  git = "https://github.com/georust/geo" }
```

Comment:
Not a big deal, but we might want to start a workspace so we can consolidate some of these dependency declarations. Or is there value in keeping them separate?

Reply:
#110 to track